src/encoded/types/file.py

```diff
@@ -1159,15 +1180,35 @@ def download(context, request):
                 properties = extra
                 external = context.propsheets.get('external' + eformat.get('uuid'))
                 if eformat is not None:
-                    tracking_values['file_format'] = eformat.get('file_format')
+                    file_format_name = eformat.get('file_format')
+                    #tracking_values['file_format'] = eformat.get('file_format')
                 break
         if not found:
             raise HTTPNotFound(_filename)
     else:
         external = context.propsheets.get('external', {})
         if file_format is not None:
-            tracking_values['file_format'] = file_format.get('file_format')
-    tracking_values['filename'] = filename
+            file_format_name = file_format.get('file_format')
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [local variable 'file_format_name' is assigned to but never used](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4885443)